versions.tf

```diff
@@ -1,13 +1,21 @@
 # Terraform version
 terraform {
-  required_version = ">= 1.0.0"
+  required_version = ">=1.6.6"
 }
 
 terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">=2.90.0"
+      version = ">=3.112.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 2.0"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 2.0"
```

Comment:
Can we use `>=` in both random and null provider?

Reply:
yess have done that